docs/assets/snippet-styles.css

```diff
@@ -33,27 +33,27 @@ It breaks CKEditor 5 (see how <p><code>[]</code></p> looks). */
 	display: none !important; /* Firefox is very stubborn. */
 }
 
-.ck-reset.ck-list {
+.ck.ck-list {
 	/* See https://github.com/ckeditor/ckeditor5/issues/494 */
 	margin-left: 0;
 }
 
 /* examples/builds/inline-editor.html */
-.live-snippet .image-style-left, .live-snippet .image-style-right {
```

Comment:
These changes don't make much sense to me. Either `.ck` or `.live-snippet`. The latter was used so far to bump up the specificity. Now, one of these become unnecessary.

Reply:
Also, `.ck` is part of the UI, and `.image-style-left` of the content. So, if anything, the selector which makes the most sense here is `.ck-content .image-style-left`. But then, the content will not look good before the editor is initialised, so it'd be good to use a class which is always present. Which is... `.live-snippet` ;)

Reply:
Let's stick to `.l-s` :P